### ISSUE TITLE
Install failure on Ubuntu 16.04: Failed to fetch ... Received HTTP code 403 from proxy after CONNECT

### ISSUE THREAD

**Issue by morrismichaelc** [asbru-cm/asbru-cm#803]
I don't have a local proxy, so I assume the proxy mentioned in the error is on the "cloudsmith" side.

While the root error seems to be:
```
Failed to fetch https://dl.cloudsmith.io/public/asbru-cm/release/deb/ubuntu/dists/xenial/main/source/Sources  Received HTTP code 403 from proxy after CONNECT
```
a fetch of that URL (https://dl.cloudsmith.io/public/asbru-cm/release/deb/ubuntu/dists/xenial/main/source/Sources) in a browser results what appears to be a normal result, with package names, versions, checksums, etc. for Package "asbru-cm" with the standard HTTP result code of "200".

I'm not sure where else to look for this issue, and found nothing here in the wiki... any help much apprecaited; Asbru-cm looks like it may be just what I've been looking for!

Thanks in advance,

Mike

FULL ERROR:
```
Fetched 335 kB in 4s (80.4 kB/s)
Reading package lists...
W: The repository 'https://dl.cloudsmith.io/public/asbru-cm/release/deb/ubuntu xenial Release' does not have a Release file.
E: Failed to fetch https://dl.cloudsmith.io/public/asbru-cm/release/deb/ubuntu/dists/xenial/main/source/Sources  Received HTTP code 403 from proxy after CONNECT
E: Some index files failed to download. They have been ignored, or old ones used instead.
 ^^^^: ... Failed to update via apt-get update - Context above.
 !!!!: Oh no, your setup failed! :-( ... But we might be able to help. :-)
 !!!!: 
 !!!!: You can contact Cloudsmith (support@cloudsmith.io) for further assistance.
 !!!!: 
```

**Comment by gfrenoy** [comment]
Strange, it does not look like something we can fix on our side.  It's a network issue.  Did you try to contact the Cloudsmith team, as suggested ?

As an alternative, you can try our legacy repository on packagecloud:

`
sudo apt-add-repository multiverse
sudo apt install curl
curl -s https://packagecloud.io/install/repositories/asbru-cm/asbru-cm/script.deb.sh | sudo bash
sudo apt install asbru-cm
`

Should still work...

**Comment by morrismichaelc** [comment]
Thanks for the tip. I did get this to work, eventually. Pretty clearly there is some issue on my side, since I did get the same `Received HTTP code 403 from proxy after CONNECT` message. However, I just now got asbru CM installed by:

1. changing the repo definition from http**S**://packagecloud.io/.... to plain http://packagecloud.io/....
2. This was necessary even after re-installing apt-transport-https (may have that name wrong, but you get the idea :-)
3. Then 'sudo apt update' ran without error, but...
4. Then asbru-cm failed to install for unmet dependency on "libgtk3-simplelist-perl", which was not available in the standard ubuntu repos for 16.04
5. So I downloaded a .deb of it from packages.debian.org which installed w/o error
6. Then the install was fine.

So, happy ending, but thought you might be interested in #4 above; maybe a documentation note, or even include the dependency? _(Though I'm aware 16.04 is no longer mainstream support...)_

Thanks again for the pointer to the packagecloud repo. I've just started asbru and defined a connection and so far it looks great - thank you for the great contribution to the community!
